src/benchmarks/micro/libraries/System.Numerics.Tensors/Perf_SignedIntegerTensorPrimitives.cs

```diff
@@ -0,0 +1,128 @@
+        public void LeadingZeroCount() => TensorPrimitives.LeadingZeroCount<T>(_source1, _destination);
+
+        [Benchmark]
+        public void OnesComplement() => TensorPrimitives.OnesComplement<T>(_source1, _destination);
+
+        [Benchmark]
+        public void PopCount() => TensorPrimitives.PopCount<T>(_source1, _destination);
+
+        [Benchmark]
+        public void ShiftLeft() => TensorPrimitives.ShiftLeft<T>(_source1, ShiftAmount, _destination);
+
+        [Benchmark]
+        public void ShiftRightArithmetic() => TensorPrimitives.ShiftRightArithmetic<T>(_source1, ShiftAmount, _destination);
+
+        [Benchmark]
+        public void ShiftRightLogical() => TensorPrimitives.ShiftRightLogical<T>(_source1, ShiftAmount, _destination);
+
+        [Benchmark]
+        public void RotateLeft() => TensorPrimitives.RotateLeft<T>(_source1, ShiftAmount, _destination);
+        
+        [Benchmark]
+        public void RotateRight() => TensorPrimitives.RotateLeft<T>(_source1, ShiftAmount, _destination);
+
+        [Benchmark]
+        public void TrailingZeroCount() => TensorPrimitives.TrailingZeroCount<T>(_source1, _destination);
+        #endregion
+
+        #region Binary Operations
+        [Benchmark]
+        public void BitwiseAnd_Vector() => TensorPrimitives.BitwiseAnd<T>(_source1, _source2, _destination);
+
+        [Benchmark]
+        public void BitwiseAnd_Scalar() => TensorPrimitives.BitwiseAnd(_source1, _scalar1, _destination);
+
+        [Benchmark]
+        public void BitwiseOr_Vector() => TensorPrimitives.BitwiseOr<T>(_source1, _source2, _destination);
+
+        [Benchmark]
+        public void BitwiseOr_Scalar() => TensorPrimitives.BitwiseOr(_source1, _scalar1, _destination);
+
+        [Benchmark]
+        public void Divide_Vector() => TensorPrimitives.Divide<T>(_source1, _ones, _destination);
+
+        [Benchmark]
+        public void Divide_Scalar() => TensorPrimitives.Divide(_source1, T.One, _destination);
+
+        [Benchmark]
+        public void Max_Vector() => TensorPrimitives.Max<T>(_source1, _source2, _destination);
+
+        [Benchmark]
+        public void Max_Scalar() => TensorPrimitives.Max(_source1, _scalar1, _destination);
+
+        [Benchmark]
+        public void MaxMagnitude_Vector() => TensorPrimitives.MaxMagnitude<T>(_source1, _source2, _destination);
+
+        [Benchmark]
+        public void MaxMagnitude_Scalar() => TensorPrimitives.MaxMagnitude(_source1, _scalar1, _destination);
+
+        [Benchmark]
+        public void Min_Vector() => TensorPrimitives.Min<T>(_source1, _source2, _destination);
+
+        [Benchmark]
+        public void Min_Scalar() => TensorPrimitives.Min(_source1, _scalar1, _destination);
+
+        [Benchmark]
+        public void MinMagnitude_Vector() => TensorPrimitives.MinMagnitude<T>(_source1, _source2, _destination);
+
+        [Benchmark]
+        public void MinMagnitude_Scalar() => TensorPrimitives.MinMagnitude(_source1, _scalar1, _destination);
```

Comment:
We can reduce these to just one for vector and one for scalar, e.g. MaxMagnitude_Vector and Min_Scalar.